### PR TITLE
N21-1967 Adjust Filestorage for instance files

### PR DESCRIPTION
**Pull request by MarvinOehlerkingCap**
# Description

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/N21-1967
- https://github.com/hpi-schul-cloud/nuxt-client/pull/3280
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/5051

## Approval for review

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by arnegns — APPROVED** [review]

File: helpers/files-storage.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ async function filesStorageInit(schoolId, parentId, parentType, readonly, req) {
 	return { filesStorage };
 }
 
-const getFileDownloadPath = (file) => `/api/v3/file/download/${file.id}/${encodeURIComponent(file.name)}`;
+const getFileDownloadPath = (file) => `/api/v3/file/download/${file.id}`;
```
Comment (SevenWaysDP) [inline helpers/files-storage.js:57]: please revert

File: static/scripts/files-storage.js [hunk 2]
```diff
@@ -270,7 +270,7 @@ $(document).ready(() => {
 	$('button[data-method="download"]').on('click', (e) => {
 		const fileRecordId = $(e.currentTarget).attr('data-file-id');
 		const fileName = $(e.currentTarget).attr('data-file-name');
-		const url = getFileDownloadUrl(fileRecordId, fileName);
```
Comment (SevenWaysDP) [inline static/scripts/files-storage.js:273]: please revet it

File: static/scripts/files-storage.js [hunk 3]
```diff
@@ -350,7 +350,7 @@ $('.videostop').on('keypress', (e) => {
 window.fileViewer = function fileViewer(type, name, id) {
 	$('#my-video').css('display', 'none');
 	let win;
-	const src = getFileDownloadUrl(id, name);
+	const src = getFileDownloadUrl(id);
```
Comment (SevenWaysDP) [inline static/scripts/files-storage.js:353]: please revert it

File: static/scripts/helpers/storage.js [hunk 4]
```diff
@@ -5,6 +5,6 @@ export const session = storageFactory(() => sessionStorage);
 
 export const apiV3FileStorageBasePath = '/api/v3/file';
 
-export function getFileDownloadUrl(id, name) {
-	return `${apiV3FileStorageBasePath}/download/${id}/${encodeURIComponent(name)}`;
+export function getFileDownloadUrl(id) {
+	return `${apiV3FileStorageBasePath}/download/${id}`;
```
Comment (SevenWaysDP) [inline static/scripts/helpers/storage.js:9]: please revert it

**Review by arnegns — APPROVED** [review]

**Review by SevenWaysDP — APPROVED** [review]

**Review by SevenWaysDP — APPROVED** [review]

**Pull request merged**